hooks/pre_validations/31-mongo_storage_engine.rb

```diff
@@ -0,0 +1,41 @@
+MONGO_DIR = '/var/lib/mongodb/'
+pulp_host_param = param('katello', 'pulp_db_seeds') || param('foreman_proxy_content', 'pulp_db_seeds')
+fail_and_exit('Unknown pulp connection') unless pulp_host_param
+MONGO_HOST = pulp_host_param.value
```

Comment:
Since this is only used in one place I'd avoid the use of a global constant.

Reply:
Updated.